HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -58,6 +58,13 @@ def customiseFor14833(process):
                 producer.includeME0 = cms.bool(False)
     return process
 
+def customiseFor16670(process):
+    for producer in esproducers_by_type(process, "DetIdAssociatorESProducer"):
+        if (producer.ComponentName.value() == 'HcalDetIdAssociator'):
+            if not hasattr(producer,'hcalRegion'):
+                producer.hcalRegion = cms.int32(1)
```

Comment:
@calabria this looks okay, but now you've set all HLT versions of this module to ignore HE. The value should probably be set to 2 by default, though I wonder if we ever run this in HLT for Phase2.